frontend/appflowy_flutter/lib/plugins/database/widgets/cell/desktop_row_detail/desktop_row_detail_url_cell.dart

```diff
@@ -18,13 +19,86 @@ class DesktopRowDetailURLSkin extends IEditableURLCellSkin {
     TextEditingController textEditingController,
     URLCellDataNotifier cellDataNotifier,
   ) {
-    return TextField(
+    return LinkTextField(
       controller: textEditingController,
       focusNode: focusNode,
+    );
+  }
+
+  @override
+  List<GridCellAccessoryBuilder> accessoryBuilder(
+    GridCellAccessoryBuildContext context,
+    URLCellDataNotifier cellDataNotifier,
+  ) {
+    return [
+      accessoryFromType(
+        GridURLCellAccessoryType.visitURL,
+        cellDataNotifier,
+      ),
+    ];
+  }
+}
+
+class LinkTextField extends StatefulWidget {
+  const LinkTextField({
+    super.key,
+    required this.controller,
+    required this.focusNode,
+  });
+
+  final TextEditingController controller;
+  final FocusNode focusNode;
+  @override
+  State<LinkTextField> createState() => _LinkTextFieldState();
+}
+
+class _LinkTextFieldState extends State<LinkTextField> {
+  bool _isLinkClickable = false;
```

Comment:
This class is private, so the variable does not have to be. (Can remove the underscore)